pkgs/by-name/sq/sql-migrate/package.nix

```
@@ -0,0 +1,38 @@
+{ stdenv
+, lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "sql-migrate";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "rubenv";
+    repo = "sql-migrate";
+    rev = "v${version}";
+    hash = "sha256-l9HW8L9khr+UfxlhH9z2mrzjemlqRbnnM7rPsf9SCtg=";
+  };
+
+  vendorHash = "sha256-GDM2YAqLkkmfOJShBbPpxS0Zs3zOK4yCfTCjohSGNIM=";
+
+  # Bundle release metadata
+  ldflags = [
+    # Omit the symbol table and debug information.
+    "-s"
+    # Omit the DWARF symbol table.
+    "-w"
+  ];
```

```suggestion
  ldflags = [
    "-s"
    "-w"
  ];
```
that's not worth to comment